Cargo.lock

```diff
@@ -0,0 +1,865 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Maybe this file should not be checked in

Reply:
I should delete it?

Reply:
If there is no corresponding Cargo.toml checked in, I guess yes